commands/orchestrationcommands/stackcommands/update.go

```diff
@@ -0,0 +1,167 @@
+package stackcommands
+
+import (
+	"errors"
+	"fmt"
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	osStacks "github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/openstack/orchestration/v1/stacks"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/stacks"
+	"github.com/rackspace/rack/util"
+)
+
+var update = cli.Command{
+	Name:        "update",
+	Usage:       util.Usage(commandPrefix, "update", "[--name <stackName> | --id <stackID>] [--template-file <templateFile> | --template-url <templateURL>]"),
+	Description: "Updates a specified stack",
+	Action:      actionUpdate,
+	Flags:       commandoptions.CommandFlags(flagsUpdate, nil),
+	BashComplete: func(c *cli.Context) {
+		commandoptions.CompleteFlags(commandoptions.CommandFlags(flagsUpdate, keysUpdate))
+	},
+}
+
+func flagsUpdate() []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name:  "name",
+			Usage: "[optional; required if `id` isn't provided] The stack name.",
+		},
+		cli.StringFlag{
+			Name:  "id",
+			Usage: "[optional; required if `name` isn't provided] The stack id.",
+		},
+		cli.StringFlag{
+			Name:  "template-file",
+			Usage: "[optional; required if `template-url` isn't provided] The path to template file.",
+		},
+		cli.StringFlag{
+			Name:  "template-url",
+			Usage: "[optional; required if `template-file` isn't provided] The url to the template.",
+		},
+		cli.StringFlag{
+			Name:  "environment-file",
+			Usage: "[optional] File containing environment for the stack",
+		},
+		cli.StringFlag{
+			Name:  "timeout",
+			Usage: "[optional] Stack creation timeout in minutes.",
+		},
+		cli.BoolTFlag{
+			Name:  "disable-rollback",
+			Usage: "[optional] Disable rollback on create/update failure.",
+		},
+		cli.StringFlag{
+			Name:  "parameter-file",
+			Usage: "[optional] Parameter values from file used to create the stack. This can be specified multiple times. Parameter value would be the content of the file",
```

Comment:
For long descriptions like this, let's use `strings.Join([]string{"line one", "line two"}, "\n")`.